.github/workflows/add-reviewer-pr.yml

```diff
@@ -0,0 +1,16 @@
+name: Add Reviewer PR
+on:
+  pull_request:
```

Comment:
This should be pull request target i think. So as no auth is required when PR raised from fork.